src/openforms/js/components/formio_builder/builder.js

```diff
@@ -450,7 +461,6 @@ const FormIOBuilder = ({
       form={formRef.current}
       options={builderOptions}
       onChange={formSchema => onChange(cloneDeep(formSchema))}
-      onUpdateComponent={() => (previousLiteralsRef.current = {})}
```

Comment:
this fires more often than expected/anticipated and it seems to work without :shrug: